src/common/hooks/useCalculateTaxesRegion.ts

```diff
@@ -10,15 +10,17 @@
 
 import { useResolveCountry } from './useResolveCountry';
 
-export function useCalculateTaxesRegion(countryId: string | number): boolean {
+export function useCalculateTaxesRegion() {
   /**
    * Supported tax regions
    */
   const supportedCountries: string[] = ['AU', 'US', 'DE'];
 
   const resolveCountry = useResolveCountry();
 
-  return supportedCountries.includes(
-    resolveCountry(countryId)?.iso_3166_2 || ''
-  );
+  return (countryId: string | number) => {
+    return supportedCountries.includes(
+      resolveCountry(countryId)?.iso_3166_2 || ''
+    );
+  };
```

Comment:
@beganovich The reason for this change is to prevent the app from breaking when the group_settings configuration is turned on and there is an attempt to switch to company settings. The direct invocation of the hook in the code is the cause of the app breaking. I encountered a console error that pertains to one of React's rules about directly calling hooks in the code. As a solution, I've converted it into a regular hook.